docs/helm-chart/using-additional-containers.rst

```diff
@@ -0,0 +1,35 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+Using additional containers
+----------------------------
+
+If you are using your own sidecar container, you can add it through the ``extraContainers`` value.
+You can define different containers for scheduler, webserver and worker pods.
+
+For example, a sidecar that syncs DAGs from object storage.
```

Comment:
Can you add little more docs tto manage-dags-files.rst also? I think it will be very helpful to understand how to use this feature.

Reply:
I think that can be done in separate PR since this just adds abilities to add extraContainers and sync'ing dags is one of the example.